tests/unit/nupic/algorithms/knn_classifier_test.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```

Comment:
There is not any unit test for KnnClassifier, only an integration test for test it together with PCA.